.github/workflows/knative-dependabot.yaml

```diff
@@ -0,0 +1,38 @@
+name: dependabot - hack scripts 
```

Comment:
This should be added to either actions or .github repo so that we can just sync the file across repos. Not everyone uses dependabot but there are a few that do and would benefit from it.

Reply:
Okay, I was hesitated to adding the common dependabot because some repo has their own dependabot like https://github.com/knative-extensions/eventing-kafka-broker/blob/main/.github/workflows/build-dependabot-pr.yaml
But I will add it to the common one.